handbook/product/roadmap.md

```diff
@@ -74,7 +74,28 @@ See [Distribution roadmap](https://github.com/sourcegraph/about/pull/1104).
 
 ## Search
 
-TO DO
+1. 🔄 Enterprise homepage
+1. 🔄 Search tour
+1. 🔄 Search expressions
+1. 🔄 Scaling indexed text search to 500k repositories
+1. 🔄 Streaming search
+1. Code monitoring (private code monitors, no sharing)
+1. Performance at scale: follow-up to “scale indexed text search to 500k repositories”, focus and scope TBD
+1. As a user, I can create a scope of code I want to search across as a list of repositories (consolidate repogroups + search scopes)
```

Comment:
Do we want to call out the term "Graphs" here?

Reply:
While it is possible that "graphs" will be the solution we go with to solve this problem, it is not certain yet, which is why we didn't call it out here.